modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/BasicInfo.tsx

```diff
@@ -193,6 +194,12 @@ export function BasicInfo({
 				/>
 			)}
 
+			<TranslationOptionsContainer
+				isApproved={isApproved}
```

Comment:
@danvb112 I think it would be better to name this attribute as "published". So it would be like:
`published={isApproved}`. It would make more sense because of the semantic. What do you think?

Reply:
@naraandrade, yes it really makes sense, we might as well open a technical debit later to change all the places that use `isApproved` to `published`.